osi_groundtruth.proto

```diff
@@ -171,4 +172,16 @@ message GroundTruth
     // derives street geometries automatically from a map reference.
     //
     optional string model_reference = 16;
+
+    // Reference lines used by LogicalLane
+    //
+    repeated ReferenceLine reference_line = 17;
+
+    // Logical lane boundaries used by LogicalLane
+    //
+    repeated LogicalLaneBoundary logical_lane_boundary = 18;
+
+    // Logical lanes used e.g. by traffic agents
+    //
+    repeated LogicalLane logical_lane = 19;
```

Comment:
Not sure if this is the right place. 
The logical lanes and physical lanes should be the same. I.e curb is a logical and Physical lane

Reply:
Logical lanes can now reference physical lanes, so the relation between the two types is now represented in OSI.